### PR TITLE
spotify-qt, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
wip, done as part of forum post: https://discuss.haiku-os.org/t/tip-how-to-get-spotify-qt-running/9746/11

File: media-sound/spotify_qt/spotify_qt-2.4.recipe [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:qdbuscpp2xml$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. -DCMAKE_INSTALL_PREFIX=/appsDir
```
Comment (diversys) [inline media-sound/spotify_qt/spotify_qt-2.4.recipe:54]: $appsDir probably?

Comment (Begasus) [reply media-sound/spotify_qt/spotify_qt-2.4.recipe:54]: Actually even that would be incorrect as "make install" doesn't work, but probably should refer to $prefix/bin rather then $appsDir :) (but yes a typo) :)

**Comment by Begasus** [comment]
Think this should do it, not checked on x86_64 so far, would be nice if someone could check a build there :)

**Comment by diversys** [comment]
Builds fine on x86_64, but the binary should be moved to $appsDir.

**Pull request merged**